### PR TITLE
Remove empty directory on scrap command

### LINKED ISSUE #6: Scrapping the last view in a directory should remove the directory
Assume the following views:

```
views
└─ products
   │ index.blade.php
```

When executing `php artisan scrap:view products.index`, it keeps the folder `products`. Since `index.blade.php` is the last view in that folder, it should remove the folder as well.

### PR DESCRIPTION
**Pull request by patrikpihlstrom**
fixes #6

### CODE REVIEW

**Comment by patrikpihlstrom** [comment]
The two test that fail are:
Sven\ArtisanView\Tests\ViewTest::it_scraps_a_view
Sven\ArtisanView\Tests\ViewTest::it_scraps_a_view_with_dot_notation

My tests pass

**Comment by svenluijten** [comment]
Could you make the failing tests & StylcCI pass please? Some examples:
- Adhere to [PSR-2](http://www.php-fig.org/psr/psr-2/) in this repository
- Keep the code clean and readable.

File: src/View.php [hunk 1]
```diff
@@ -210,4 +214,39 @@ private function clean()
 
         return $this;
     }
+
+	/**
+	 * @param string $path $file passed from scrapView, $dir passed from removeDir.
+	 * @return bool|string false|Directory containing the file specified or the parent directory of the directory
+	 * specified.
+	 */
```
Comment (svenluijten) [inline src/View.php]: Docblock should have a short description of the method.

File: src/View.php [hunk 2]
```diff
@@ -210,4 +214,39 @@ private function clean()
...
+
+		return false;
+	}
+
+	/**
+	 * Remove the specified directory and check if the parent directory should be removed.
+	 *
+	 * @param string $dir The directory to be removed.
+	 */
+	private function removeDir($dir)
```
Comment (svenluijten) [inline src/View.php]: Don't abbreviate variable & method names.

**Comment by patrikpihlstrom** [comment]
Sure thing 👍

**Comment by svenluijten** [comment]
Looks great! Can you get [StyleCI](https://styleci.io/analyses/zDMogb) passing please?

**Comment by patrikpihlstrom** [comment]
Should work now

**Comment by svenluijten** [comment]
Thanks!

**Pull request merged**